cucumber-expressions/README.md

```diff
@@ -1,9 +1,27 @@
 See [website docs](https://cucumber.io/docs/cucumber/cucumber-expressions/) for more details.
 
+
+## Grammar ##
+
+```
+cucumber-expression :=  [ optional | escaped-optional | other-then-optional ]*
```

Comment:
Is this an indicative order of the algorithm?

Reply:
Yes. A precedence order parser first parses the highest precedence operator, then the lower precedence operators.